pom.xml

```diff
@@ -11,14 +11,22 @@
   </parent>
 
   <artifactId>jitsi-lgpl-dependencies</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>1.1-SNAPSHOT</version>
   <packaging>bundle</packaging>
 
   <name>jitsi-lgpl-dependencies</name>
   <url>https://github.com/jitsi/jitsi-lgpl-dependencies/</url>
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+            <source>1.8</source>
+            <target>1.8</target>
+        </configuration>
```

Comment:
Hey @ibauersachs why we added it as 1.8? Maybe we should fix this, we want current jitsi-desktop to be 1.7 dependent and from next release we will start with 1.8, right?

Reply:
Probably out of habit, can't remember. I don't mind which version we target for this specific release, but I probably used 1.8 features that would need to be reverted.